transplant/transplant_master.py

```diff
@@ -523,6 +525,11 @@ def __init__(self, executable='matlab', arguments=tuple(), msgformat='msgpack',
                 arguments += '-minimize',
         if not jvm and '-nojvm' not in arguments:
             arguments += '-nojvm',
+        paths = (os.path.dirname(__file__))
```

Comment:
Why is this expression parenthesized?

Reply:
The paths here is meant to be a tuple thus more paths can be added.

Reply:
Note that a tuple is created by the comma, not the parenthesis. `(1)` is identical to `1`. Only `(1,)` makes it a tuple.